src/url.rs

```diff
@@ -1,8 +1,13 @@
 use idna::punycode::decode_to_string;
+use pyo3::once_cell::GILOnceCell;
 use pyo3::prelude::*;
 use pyo3::types::PyDict;
 use url::Url;
 
+use crate::validators::SchemaValidator;
```

Comment:
```suggestion
use crate::SchemaValidator;
```

Should work I think?